apis/swagger.yml

```diff
@@ -648,6 +648,47 @@ paths:
         500:
           $ref: "#/responses/500ErrorResponse"
 
+  /peer/heartbeat:
+    get:
```

Comment:
I do not think this is a "GET" request. Maybe `POST` is better.  @wangforthinker